### ISSUE TITLE
create pipes cannot pull from backpacks with tanks

### ISSUE THREAD

**Issue by Quezler** [NillerMedDild/Enigmatica6#3292]
### Modpack Version

0.5.11

### Describe your issue.

originally discovered in the [go create](https://www.curseforge.com/minecraft/modpacks/go-create) modpack since there are no other fluid pipes, but i have no idea which of the 2 mods (sophisticated backpacks or create) i should point the blame at, and it's reproducible in enigmatica 6 as well 🙃 

basically the create fluid system claims it can pipe from any block with fluid capabilities:

![Screen Shot 2021-09-21 at 19 29 08](https://user-images.githubusercontent.com/3179271/134219420-9abf7caf-73ce-4704-beeb-0f555de8857f.png)

however it does not seem to be able to pipe liquids from the backpack, whilst other mod pipes can do it just fine:

![2021-09-21_19 29 23](https://user-images.githubusercontent.com/3179271/134219435-96a48166-a1ce-4ef2-b861-860d38e31ea9.png)
![2021-09-21_19 29 35](https://user-images.githubusercontent.com/3179271/134219443-1fc7ba6b-6d02-4315-9e5c-8be024022ff8.png)

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Both

**Comment by Quezler** [comment]
not entirely sure how to interpret what exactly you're asking, but here is a setup with backpacks & pipes:

https://discord.com/channels/195234321836146688/726178356143456287/874988929365991504

**Comment by NillerMedDild** [comment]
I'll rephrase: The Create setup you depicted in the report, can that be used as-is to fill Create tanks?

**Comment by Quezler** [comment]
yes, the issue appears to only be the extraction of the backpack, the pipe isn't even visibly filling up 🤔
